module/BingoArea.h

```diff
@@ -0,0 +1,83 @@
+/**
+ * @file BingoArea.h
+ * @brief ビンゴエリアの情報を保持するクラス
+ * @author Hisataka-Hagiyama,uchyam
+ */
+
+#ifndef BINGOAREA_H
+#define BINGOAREA_H
+
+#include "array"
+#include "BlockCircle.h"
+#include "CrossCircle.h"
+#include "CenterMark.h"
+#include "LineTraceArea.h"
+#include "Measurer.h"
+
+class BingoArea {
+ public:
+  //コンストラクタ
+  //交点サークル、ブロックサークルについて座標、サークルの色を設定、交点サークルの黒線の本数を設定
+  BingoArea();
+
+  /**
+   *@fn Node& getNode(int blockId);
+   *@brief　ブロック番号を指定し、ノードの情報を取得する
+   *@param blockId ブロック番号
+   *@return Nodeの参照
+   */
+  Node& getNode(BLOCK_ID blockId);
+
+  /**
+   *@fn BlockCircle& getBlockCircle(CIRCLE_ID circleId);
+   *@brief　ブロックサークルIDを指定し、ブロックサークルの情報を取得する
+   *@param circleId ブロック番号

```

Comment:
```suggestion
   *@param circleId ブロックサークルID

```
かな？
ここってIDをそのままint型にcastしてるだけみたいだし，getCrossCircleと同じようにindexでもいいんじゃないって思ったんだけど，何が違うの？
blockCircleのインデックスとcircleIdの違いが良く分からない